noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -2434,10 +2612,16 @@
                 ]]>
             </info>
         </queries>
-    </rule>
+    </check>
+
+    <!-- Tests -->
 
-    <rule>
+    <test id="ELT1">
         <title>Check the references in endringslogg</title>
+        <description>
+            Tests whether all objects referenced in the change log can be found
```

Comment:
`Tests whether all objects referenced in the change log can be found by their systemID in arkivstruktur.xml`